src/__tests__/pages/index.test.js

```diff
@@ -1031,6 +1032,27 @@ describe('index.js', () => {
     expect(notification.exists()).toBe(true)
   })
 
+  it('does not render a SFAC extension notification if showSfacExtensionPrompt is not true', () => {
+    const IndexPage = require('src/pages/index').default
+    const mockProps = getMockProps()
+    useData.mockReturnValue({ data: mockProps.data })
+    const wrapper = mount(<IndexPage {...mockProps} />)
+
+    const notification = wrapper.find(SfacExtensionSellNotification)
+    expect(notification.exists()).toBe(false)
+  })
+
+  it('does render a SFAC extension notification if showSfacExtensionPrompt is true', () => {
+    const IndexPage = require('src/pages/index').default
+    const mockProps = getMockProps()
+    mockProps.data.user.showSfacExtensionPrompt = true
+    useData.mockReturnValue({ data: mockProps.data })
+    const wrapper = mount(<IndexPage {...mockProps} />)
+
+    const notification = wrapper.find(SfacExtensionSellNotification)
+    expect(notification.exists()).toBe(true)
+  })
+
```

Comment:
Add tests for supported/unsupported browsers.

Reply:
Ah yes, good call.